packages/babel-register/README.md

```diff
@@ -47,18 +47,22 @@ require("@babel/register")({
 
 ```javascript
 require("@babel/register")({
-  // Optional ignore regex - if any filenames **do** match this regex then they
-  // aren't compiled.
-  ignore: [/regex/],
-
-  // Ignore can also be specified as a function.
-  ignore: [function(filename) {
-    if (filename === "/path/to/es6-file.js") {
-      return false;
-    } else {
+  // Optional ignore array
```

Comment:
Maybe you could add something to make it clear that the file will be ignored if it matches any of the `ignore` entries?